docs/DeveloperGuide.md

```diff
@@ -308,6 +342,180 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Edit a person's details**
+
+**MSS**
+
+1.  User requests to list persons
+2.  Connections show a list of persons
+3.  User provides his index in the list along with the replacement information
+4.  Connections reflect the edits that were made
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+    
+    * 3a1. Connections show an error message.
+      
+      Use case resumes at step 2.
+
+**Use case: Exit the program**
+
+**MSS**
+
+1.  User request to exit
+2.  Connections exit
+
+    Use case ends.
+
+**Extensions**
+
+**Use case: Find a person**
+
+**MSS**
+
+1.  User chooses to look for an entry
+2.  Use provides the search term
+3.  Connections return all entries that matches the search term
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. Connections is empty
+
+    * 2a1. Connections display a message to indicate no matching entries
```

Comment:
Maybe display there are no entries is better to differentiate from here and the case below?